frontend/packages/kubevirt-plugin/src/selectors/selectors.ts

```diff
@@ -7,29 +7,33 @@ export const getKind = (value) => _.get(value, 'kind') as K8sResourceKind['kind'
 export const getGeneratedName = (value) =>
   _.get(value, 'metadata.generateName') as K8sResourceKind['metadata']['generateName'];
 
-export const getLabels = (entity: K8sResourceKind, defaultValue?: any) =>
-  _.get(entity, 'metadata.labels', defaultValue) as K8sResourceKind['metadata']['labels'];
-export const getAnnotations = (vm: VMGenericLikeEntityKind, defaultValue?: any) =>
-  _.get(vm, 'metadata.annotations', defaultValue);
-export const getAnnotation = (
-  vm: VMGenericLikeEntityKind,
-  annotationName: string,
-  defaultValue?: any,
-) => _.get(vm, ['metadata', 'annotations', annotationName], defaultValue);
-
 export const getDescription = (vm: VMGenericLikeEntityKind) =>
   _.get(vm, 'metadata.annotations.description');
 
-export const getLabelValue = (entity: K8sResourceKind, label: string): string =>
-  _.get(entity, ['metadata', 'labels', label]);
-
 export const getStorageSize = (value): string => _.get(value, 'requests.storage');
 
 export const getValueByPrefix = (obj = {}, keyPrefix: string): string => {
   const objectKey = Object.keys(obj).find((key) => key.startsWith(keyPrefix));
   return objectKey ? obj[objectKey] : null;
 };
 
+// Labels
```

Comment:
this should probably go to console/shared, but I am okay with doing this later.

Reply:
Nevertheless it should have better typing

Reply:
What in particular are you concerned about in terms of typing?

Reply:
`defaultValue?: string` and better return types

Reply:
Done!